### PR TITLE
Amit drivetrain

### PR DESCRIPTION
**Pull request by AmitBrunicki**

### CODE REVIEW

File: src/main/java/frc/robot/OI.java [hunk 1]
```diff
@@ -4,11 +4,10 @@
 
 public class OI /*GEVALD*/ {
 
-    private Joystick left = new Joystick(0);
-    private Joystick right = new Joystick(1);
+    Joystick left = new Joystick(0);
```
Comment (OfriRosenbaum) [inline src/main/java/frc/robot/OI.java]: where private?

File: src/main/java/frc/robot/OI.java [hunk 2]
```diff
@@ -4,11 +4,10 @@
...
+    Joystick right = new Joystick(1);
```
Comment (OfriRosenbaum) [inline src/main/java/frc/robot/OI.java]: ditto

File: src/main/java/frc/robot/subsystems/Drivetrain.java [hunk 3]
```diff
@@ -10,26 +10,43 @@
...
     public static Drivetrain getInstance() {
         if (instance == null) {
-            instance = new Drivetrain();
+            instance = new Drivetrain("drivetrain",
+                    new CANSparkMax(RobotMap.CAN.DRIVETRAIN_LEFT_SPARKMAX_1, CANSparkMaxLowLevel.MotorType.kBrushless),
+                    new CANSparkMax(RobotMap.CAN.DRIVETRAIN_LEFT_SPARKMAX_2, CANSparkMaxLowLevel.MotorType.kBrushless),
+                    new CANSparkMax(RobotMap.CAN.DRIVETRAIN_RIGHT_SPARKMAX_1, CANSparkMaxLowLevel.MotorType.kBrushless),
+                    new CANSparkMax(RobotMap.CAN.DRIVETRAIN_RIGHT_SPARKMAX_2, CANSparkMaxLowLevel.MotorType.kBrushless)
+            );
+
```
Comment (OfriRosenbaum) [inline src/main/java/frc/robot/subsystems/Drivetrain.java]: remove blank line

**Review by OfriRosenbaum — CHANGES_REQUESTED** [review]

**Review by OfriRosenbaum — APPROVED** [review]

File: src/main/java/frc/robot/subsystems/Drivetrain.java [hunk 4]
```diff
@@ -10,26 +10,42 @@
...
-    
-    private ADXRS450_Gyro gyro;
-        
+
+    private final CANSparkMax left1;
+    private final CANSparkMax left2;
+    private final CANSparkMax right1;
+    private final CANSparkMax right2;
+
+    private final ADXRS450_Gyro gyro;
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/frc/robot/subsystems/Drivetrain.java]: pigeon imu instead

**Review by Yoel-Perman-Brilliant — CHANGES_REQUESTED** [review]

**Review by OfriRosenbaum — APPROVED** [review]

**Pull request merged**